app/controllers/activities_controller.rb

```diff
@@ -30,4 +30,36 @@ def poll
       format.js
     end
   end
+
+  private
+
+  def filtering_params
+    params.permit(:user_id, :trackable_type, :since, :before)
+  end
+
+  def filter_activities(activities)
+    if params[:since].present? && valid_date?(params[:since])
+      activities = activities.since(DateTime.parse(params[:since]).beginning_of_day)
```

Comment:
We don't need `beginning_of_day` since DateTime.parse on a date like `2024-08-01` will return beginning of day: `Thu, 01 Aug 2024 00:00:00 +0000`